components/director/internal/model/api.go

```diff
@@ -13,9 +16,9 @@ import (
 )
 
 type APIDefinition struct {
-	ApplicationID                           string
-	PackageID                               *string
-	Tenant                                  string
+	ApplicationID                           string  `hash:"ignore"`
```

Comment:
Why are these tags put here as you never hash the "model" struct but only the "input" one? Same for Events/Packages.

From my POV only the `Version` value in the `Input` struct should be ignored.

Reply:
These should have been leftovers as previously we hashed the db model as well for comparison, do you remember